.github/ISSUE_TEMPLATE.md

```diff
@@ -1,34 +1,20 @@
-
 # Before you proceed
 
 - Read the [wiki](https://github.com/ckb-next/ckb-next/wiki).
-- Try the latest code by building the software manually.
 - Search in open and closed [Issues](https://github.com/ckb-next/ckb-next/issues).
+- If possible, try the latest code by building the software manually.
 - File unrelated issues separately.
 - Use [Markdown](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet) to format the message.
-- There are 3 types of issues: new device, feature, bug. Use only one. Delete redundant sections before sending.
-
+- There are 3 types of issues: new device, feature, bug.
+Use only one. Delete redundant sections before sending.
+If your issue doesn't fit in any of those categories, please use the *Something else* field at the bottom.
+- **Click the `Generate Report` button in the `Settings` tab and attach the generated report to this issue.**
```

Comment:
I actually clicked the "settings" section on this page because that's the kind of guy I am. Maybe we can attach a mini-screenshot advocating the place and clickability of said button?

Reply:
We could but the problem with that is that unless they switch to preview, they won't really see it.